docs/UserGuide.adoc

```diff
@@ -674,6 +673,13 @@ Goes to the Online tab which is a web view.
 * `tab 3` +
 Goes to the Offline tab which shows the offline copy of bookmarks.
 
+
+=== Autocomplete commands `[coming in v2.0]`
+Auto-completes search for commands line if the command is unique.
```

Comment:
```suggestion
Automatically fills in partially typed commands if the command is unique.
```

I am not sure about this but personally I feel such phrasing is clearer.

Reply:
Fixed